validator/coordinator/src/main/java/tech/pegasys/artemis/validator/coordinator/ValidatorCoordinator.java

```diff
@@ -83,6 +92,10 @@ private void simulateNewMessages() {
               state.getSlot().plus(UnsignedLong.ONE), blockRoot, MockStateRoot, deposits);
 
       BLSSignature epoch_signature = setEpochSignature(state);
+      // If the signature is missing it isnt our validator
```

Comment:
Shouldn't it be, if signature is missing, it is our validator?

Reply:
i'm not totally sure what to do here.  i have the flu and my brain is fuzzy

Reply:
The block is being created right then though. The intention was to create the block and sign it if it is ours